packages/core/src/lib/connection_manager.ts

```diff
@@ -302,25 +319,16 @@ export class ConnectionManager
           Tags.BOOTSTRAP
         );
 
-        if (isBootstrap) {
-          this.dispatchEvent(
-            new CustomEvent<PeerId>(
-              EPeersByDiscoveryEvents.PEER_DISCOVERY_BOOTSTRAP,
-              {
-                detail: peerId,
-              }
-            )
-          );
-        } else {
-          this.dispatchEvent(
-            new CustomEvent<PeerId>(
-              EPeersByDiscoveryEvents.PEER_DISCOVERY_PEER_EXCHANGE,
-              {
-                detail: peerId,
-              }
-            )
-          );
-        }
+        this.dispatchEvent(
+          new CustomEvent<PeerId>(
+            isBootstrap
+              ? EPeersByDiscoveryEvents.PEER_DISCOVERY_BOOTSTRAP
```

Comment:
I am wondering what this `E` in front means?

Reply:
I am facing it the first time. I think it is better to mention explicitly `Enum` in the name